.github/workflows/test.yml

```diff
@@ -270,13 +271,21 @@ jobs:
     - uses: actions/checkout@v3
       name: Checkout TTK source code
 
+    - name: Remove hosted Python
+      run: |
+        sudo rm -rf /Library/Frameworks/Python.framework
+        sudo rm -rf /usr/local/Frameworks/Python.framework
+
+
     - name: Install macOS dependencies
       run: |
         # ParaView dependencies
+        brew update
+        brew reinstall python@3.11
```

Comment:
the current version of the repo simply includes `brew reinstall python`

Reply:
Same as above